app/controllers/api/mixins/resource_cancel.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  module Mixins
+    module ResourceCancel
```

Comment:
Is this ResourceCancel or TaskCancel?

Reply:
It's for `ServiceOrder`, `MiqRequest`X4 or `MiqRequestTask` primary collections.